cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -138,15 +139,23 @@ func (i *deployCommand) RegisterHook() {
 			if err := CheckDeployArgs(&i.flags.CommonFlags); err != nil {
 				return err
 			}
+			if err := common.CheckChannelArgs(&i.flags.ChannelFlags); err != nil {
+				return err
+			}
 			return nil
 		},
 	}
 }
 
 func (i *deployCommand) InitHook() {
-	i.flags = deployFlags{CommonFlags: CommonFlags{}}
+	i.flags = deployFlags{
+		CommonFlags:  CommonFlags{},
+		ChannelFlags: common.ChannelFlags{},
+	}
 	i.Parent.AddCommand(i.command)
 	AddDeployFlags(i.command, &i.flags.CommonFlags)
+	common.AddChannelFlags(i.command, &i.flags.ChannelFlags)
```

Comment:
By composing the `OperatorFlags` to `deployFlags` you wouldn't need to this.

Reply:
done